5.5/run-mysqld-slave.sh

```diff
@@ -6,6 +6,7 @@ source ${HOME}/common.sh
 set -eu
 
 export MYSQL_RUNNING_AS_SLAVE=1
+export MYSQL_MASTER_ADDR=${MYSQL_MASTER_ADDR:-mysql-master}
```

Comment:
since it's just a dns lookup for "mysql-master" i don't see how it would know to specifically search the current namespace unless that's part of the /etc/resolv.conf that's mounted into the pod?

the fact that it's not provided means this image is dependent on the service existing with exactly that name..that seems bad.  it used to be a variable based on the service name......